src/epic.rs

```diff
@@ -302,6 +302,665 @@ pub static EPICS: &[Epic] = &[
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Battlefield Where The Moon Says I Love You",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Emperor Shaka the Great",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Lay of the Children of Húrin",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Lay of Leithian",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Giannina Braschi's Empire of Dreams",
+        wiki: Wiki::Name("Braschi's Empire of Dreams"),
+    },
+    Epic {
+        title: "Omeros",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Arundhati",
+        wiki: Wiki::Name("Arundhati (epic)"),
+    },
+    Epic {
+        title: "Mastorava",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Astronautilía Hvězdoplavba",
+        wiki: Wiki::Name("Astronautilia"),
+    },
+    Epic {
+        title: "Fredy Neptune: A Novel in Verse",
+        wiki: Wiki::Name("Fredy Neptune"),
+    },
+    // 21st century
+    Epic {
+        title: "Sribhargavaraghaviyam",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Ashtavakra",
+        wiki: Wiki::Name("Ashtavakra (epic)"),
+    },
+    Epic {
+        title: "Gitaramayanam",
+        wiki: Wiki::Same,
+    },
+    // Other epics
+    Epic {
+        title: "Gesta Berengarii imperatoris",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Epic of Bamana Segu",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Epic of Darkness",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Epic of Jangar",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Epic of Köroğlu",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Epic of the Forgotten",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Hikayat Seri Rama",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Hinilawod",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Khun Chang Khun Phaen",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Klei Khan Y Dam San",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Koti and Chennayya",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Epic of Siri",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Kutune Shirka",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Parsifal",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Pasyón",
+        wiki: Wiki::Name("Pasyon"),
+    },
+    Epic {
+        title: "Ramakien",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Der Ring des Nibelungen",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Siribhoovalaya",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Sundiata",
+        wiki: Wiki::Name("Epic of Sundiata"),
+    },
+    Epic {
+        title: "Yadegar-e Zariran",
```

Comment:
This appears to be a repeat of lines 302 https://github.com/a2aaron/mario-epics/blob/75fe16cfe64105f8636c35de92a1d1d76a398b81/src/epic.rs#L301-L304